accounts/accounts.go

```diff
@@ -0,0 +1,54 @@
+package accounts
+
+import "errors"
+
+// Account : State of a particular account
```

Comment:
`gometalinter` complains if I don't have comments for exported stuff. It probably knows best.